flowbased-computation/flowbased-computation-api/pom.xml

```diff
@@ -38,8 +38,9 @@
             <artifactId>powsybl-sensitivity-api</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.powsybl</groupId>
-            <artifactId>powsybl-tools</artifactId>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>farao-commons</artifactId>
```

Comment:
For me it compiles without this dependency but I'm not sure to understand why

Reply:
ok removed